package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gasbuddy/configured-etcd-client",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I think this should be a major version bump

Reply:
They are backwards compatible changes though 😛